microsoft/electionguard-python
# 📝 Add Sample Election Manifest

### Issue
Fixes #80 

Adds a sample election manifest and a generator to simulate ballots for the manifest

### To Test
1. in `electionguardtest/sample_generator.py` set `DEFAULT_NUMBER_OF_BALLOTS` to something reasonable (like 5 or 10)
2. run `make generate-sample-data`
3. expected: the election results are output, along with a `private` folder that includes objects with secrets
